libs/blocks/caas-config/caas-config.css

```diff
@@ -36,3 +36,72 @@ dd.content .sort-options > .field {
   top: 12px;
 }
 
+.config-panel dd.content.is-expanded {
+  max-height: 6000px;
+}
+
+.multifield .multifield {
+  padding: 0 0 15px;
+  margin: 0 -15px 9px 0;
+  border-bottom: solid 1px #999;
+  border-bottom: none;
+}
+
+.multifield .multifield > .multifield-set {
+  border: solid 1px #999;
+  padding: 3px 6px;
+  border-radius: 4px;
+  background: #757575;
+}
+
+.multifield.filtersCustom > .multifield-set:nth-child(2) {
+  margin-top: -30px;
+}
+
+.multifield .multifield > .multifield-set:not(:nth-child(2)) {
+  border-top: none;
+}
+
+.config-panel dd.content .multifield .multifield .field {
+   margin-bottom: 0;
+}
+
+.multifield .multifield h3 {
+  font-size: 16px;
+  font-weight: 300;
+  margin: 0;
+}
+
+.multifield .multifield > .multifield-set > .multifield-fields {
+  width: 190px;
```

Comment:
Fixed width is usually bad practice, as things are static rather than responsive.

Reply:
In this case the container is inside another container with a fix width and constrain dimensions, so I figure it will work since the parent element is not fluid/responsive.